app/services/foreman/renderer/scope/macros/base.rb

```diff
@@ -416,14 +416,19 @@ def foreman_server_ca_cert(server_ca_file_enabled: true, ssl_ca_file_enabled: tr
             example 'join_with_line_break(values) # => 1\n2\n3 for CSV,JSON,YAML'
           end
           def join_with_line_break(array)
+            # Normalize elements of the array to end with "\n" if they do
+            # not do already.
+            array.map! { |str| str[-1] == "\n" ? str : str + "\n" }
             case report_format.mime_type
             when 'text/csv'
-              array.join("\n")
+              array.join('')
```

Comment:
Avoids repeating `.join('')`, just `.join` works the same.
```suggestion
              array.join
```